CHANGELOG.unreleased.md

```diff
@@ -12,7 +12,13 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - **XO 6**:
-  - [VM,Host/Console] Display _Console Clipboard_ and _Console Actions_ (PR [#8125](https://github.com/vatesfr/xen-orchestra/pull/8125))
+
+  - [VM,Host/Console] Display _Console Clipboard_ and _Console Actions_ (
+    PR [#8125](https://github.com/vatesfr/xen-orchestra/pull/8125))
+
+- **XO Lite**:
+  - [Host/Console] Display _Console Clipboard_ and _Console Actions_ (
+    PR [#8136](https://github.com/vatesfr/xen-orchestra/pull/8136))
```

Comment:
XOLite has its own changelog because it has a different release process
`@xen-orchestra/lite/CHANGELOG.md`